kratos/input_output/vtk_output.cpp

```diff
@@ -17,84 +17,82 @@
-    if(file_format == "ascii")
-        mFileFormat =  VtkOutput::FileFormat::VTK_ASCII;
-    else if (file_format == "binary")
+    mDefaultPrecision = mOutputSettings["output_precision"].GetInt();
+    const std::string file_format = mOutputSettings["file_format"].GetString();
+    if (file_format == "ascii") {
+        mFileFormat = VtkOutput::FileFormat::VTK_ASCII;
+    }
+    else if (file_format == "binary") {
         mFileFormat = VtkOutput::FileFormat::VTK_BINARY;
-    else
-        KRATOS_ERROR<<"Option for file_format : "<<file_format<<" not recognised.!"<<std::endl
-            <<"Possible output formats options for VTKOutput are :: ascii and binary "<<std::endl;
+    }
+    else {
+        KRATOS_ERROR << "Option for \"file_format\": " << file_format
+            << " not recognised!\n Possible output formats "
+            << "options are: \"ascii\", \"binary\"" << std::endl;
+    }
 }
 
-void VtkOutput::CreateMapFromKratosIdToVTKId(const ModelPart &rModelPart)
+void VtkOutput::CreateMapFromKratosIdToVTKId(const ModelPart& rModelPart)
 {
     int vtk_id = 0;
-    for(const auto& node : rModelPart.Nodes())
-    {
-        mKratosIdToVtkId[node.Id()] = vtk_id++;
+    for(const auto& r_node : rModelPart.Nodes()) {
+        mKratosIdToVtkId[r_node.Id()] = vtk_id++;
     }
 }
 
 template<typename TContainerType>
-unsigned int VtkOutput::DetermineVtkCellListSize(const TContainerType &rContainer)
+unsigned int VtkOutput::DetermineVtkCellListSize(const TContainerType& rContainer) const
 {
     unsigned int vtk_cell_list_size_container = 0;
 
     const auto container_begin = rContainer.begin();
     const int num_entities = rContainer.size();
     #pragma omp parallel for reduction(+:vtk_cell_list_size_container)
-    for (int i=0; i<num_entities; ++i)
-    {
-        auto entity_i = container_begin + i;
-        vtk_cell_list_size_container += entity_i->GetGeometry().PointsNumber() +1;
+    for (int i=0; i<num_entities; ++i) {
+        const auto entity_i = container_begin + i;
+        vtk_cell_list_size_container += entity_i->GetGeometry().PointsNumber() + 1;
     }
 
     return vtk_cell_list_size_container;
 }
 
-void VtkOutput::Initialize(const ModelPart &rModelPart)
+void VtkOutput::Initialize(const ModelPart& rModelPart)
 {
     CreateMapFromKratosIdToVTKId(rModelPart);
 }
 
-void VtkOutput::WriteHeader(const ModelPart &rModelPart, std::ofstream& rFileStream)
+void VtkOutput::WriteHeader(const ModelPart& rModelPart, std::ofstream& rFileStream) const
 {
-    rFileStream << "# vtk DataFile Version 4.0"
-               << "\n";
-    rFileStream << "vtk output"
-               << "\n";
-    if(mFileFormat == VtkOutput::FileFormat::VTK_ASCII)
-        rFileStream << "ASCII"<< "\n";
-    else if (mFileFormat == VtkOutput::FileFormat::VTK_BINARY)
-        rFileStream << "BINARY"<< "\n";
-    rFileStream << "DATASET UNSTRUCTURED_GRID"
-               << "\n";
+    rFileStream << "# vtk DataFile Version 4.0\nvtk output\n";
```

Comment:
I really like the old way .. that is one line after the other (with just \n put beside the text). This is because this is how it appears in the output file.

Reply:
sorry, I can change it back

Reply:
I would love to see it back.